src/glmfit.jl

```diff
@@ -230,6 +247,41 @@ fit{M<:AbstractGLM}(::Type{M}, X::@compat(Union{Matrix,SparseMatrixCSC}), y::Abs
                     d::UnivariateDistribution, l::Link=canonicallink(d); kwargs...) =
     fit(M, float(X), float(y), d, l; kwargs...)
 
+"""
+    glm(X, y, args...; kwargs...)
+
+Fits the generalised linear model, specified by the symbolic description of the
+model along with the family of distribution and a link function.
+
+#### Arguments:
+* `X`: The formula which is the symbolic representation of the model to fit.
+Uses column symbols from the DataFrame data, for example, if names(data)
+=[:Y,:X1,:X2], then a valid formula is Y~X1+X2
+* `y`: A DataFrame which may contain NA values, any rows with NA values are ignored.
+* `family`: Specifies the choice of variance, can be `Bernoulli()`, `Binomial()`,
```

Comment:
`family` and `link` should be added to the signature, even if they are technically passed to another function. Please also document `dofit`, `wts` and `offset`.